dags/taar_weekly.py

```diff
@@ -30,6 +47,128 @@
     "taar_weekly", default_args=default_args_weekly, schedule_interval="@weekly"
 )
 
+
+def wipe_gcs_files():
+    # This just makes sure we clear the bucket.
+    return [
+        "bash",
+        "-c",
+        # gsutil returns an error if you try to delete 0 files
+        # so we need to ignore errors here
+        """/google-cloud-sdk/bin/gsutil -m rm gs://%s/* || /bin/true"""
+        % TAAR_ETL_STORAGE_BUCKET,
+    ]
+
+
+wipe_gcs_bucket = GKEPodOperator(
+    owner="vng@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com",],
+    task_id="wipe_taar_gcs_bucket",
+    name="wipe_taar_gcs_bucket",
+    image="google/cloud-sdk:242.0.0-alpine",
+    arguments=wipe_gcs_files(),
+    location="us-central1-a",
+    cluster_name="bq-load-gke-1",
+    dag=taar_weekly,
+)
+
+dump_bq_to_tmp_table = GKEPodOperator(
+    owner="vng@mozilla.com",
+    email=["vng@mozilla.com", "mlopatka@mozilla.com",],
+    task_id="dump_bq_to_tmp_table",
+    name="dump_bq_to_tmp_table",
+    # This uses a circleci built docker image from github.com/mozilla/taar_gcp_etl
+    image=TAAR_ETL_CONTAINER_IMAGE,
+    arguments=[
+        "-m",
```

Comment:
just a nit that in the previous step you call a function to generate args and here you're inlining them

Reply:
nvm i see that you reuse these args elsewhere

Reply:
I've extracted the common arguments into a function in 3f35557